README.md

```diff
@@ -1,40 +1,77 @@
 foxsi-science
 =============
 
+INSTALL:
+
+    You need to have IDL and SSW previously installed
+
+    Include this five lines in your personal IDL_STARTUP file
```

Comment:
You should also instruct the user to change these lines to reflect their own directory structure, and tell them what to change.  For example, ~/Desktop/foxsi-science should be changed to whatever path is needed.  With the next line, is WORK_SSWIDL/Default/foxsidb something the user needs to change or is WORK_SSWIDL an environment variable that SSWIDL already knows at this point?  Give the user very specific instructions on how to change these lines.